static/js/reducers/order_receipt.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import moment from 'moment';
+
+import {
+  SET_TIMEOUT_ACTIVE,
+  SET_INITIAL_TIME,
+} from '../actions/order_receipt';
+
+export type OrderReceiptState = {
+  initialTime: string,
+  timeoutActive: boolean,
+};
+
+export const INITIAL_ORDER_RECEIPT_STATE = {
+  timeoutActive: false,
+  initialTime: moment().toISOString()
+};
```

Comment:
Yeah, this isn't ideal but I think it works fine for this particular use case, where the module load and the component mount happen pretty much the same time